recipes/barrnap/0.2/meta.yaml

```diff
@@ -0,0 +1,22 @@
+about:
+  home: https://github.com/tseemann/barrnap
+  license: GPL3
+  summary: 'Barrnap predicts the location of 5S, 16S and 23S ribosomal RNA genes in
+    Bacterial genome sequ It takes FASTA DNA sequence as input, and write GFF3 as
+    output.'
+build:
+  number: 0
+package:
+  name: barrnap
+  version: '0.2'
+requirements:
+  build: []
```

Comment:
Nothing official yet. Since it is not affecting the packages for the user, it is also not critical. I always try to have a minimal description (no unnecessary lines).

Apart from that, now that I look at the recipe again, I would prefer to not have a strict perl 5.22.0 dependency here. What happens if we or the user wants to update perl? I (and Conda philosophy) see dependencies as technical requirements, and technically barrnap should still be usable. Reproducibility can be achieved via meta-packages, environments and flat lists as discussed in the other pull request.